tests/test_fixtures/rule_based_profiler/example_notebooks/DataAssistants_Instantiation_And_Running-OnboardingAssistant-Spark.ipynb

```diff
@@ -0,0 +1,1491 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "834eb3b0",
+   "metadata": {},
+   "source": [
+    "# How to use `DataAssistants`\n",
+    "\n",
+    "* A `DataAssistant` enables you to quickly profile your data by providing a thin API over a pre-constructed `RuleBasedProfiler` configuration.\n",
+    "* As a result of the profiling, you get back a result object consisting of \n",
+    "    * `Metrics` that describe the current state of the data\n",
+    "    * `Expectations` that are able to alert you if the data deviates from the expected state in the future. \n",
+    "    \n",
+    "* `DataAssistant` results can also be plotted to help you understand their data visually.\n",
+    "* There are multiple `DataAssistants` centered around a theme (volume, nullity, etc), and this notebook walks you through an example `OnboardingDataAssistant`, which is the most general and extensive `DataAssistant`.\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "26b3aab6",
+   "metadata": {},
+   "source": [
+    "The `OnboardingDataAssistant` is considered to be the \"starting point\" for profiling and is generally applicable for numerical, categorial, or datetime data.  In our example we will be using `taxi_trip` data, building our `ExpectationSuite` using data from 2019, and validating the suite on January 2020 data, to see if our more-recent data falls within the range of previous months.\n",
+    "\n",
+    "In our example, the `OnboardingDataAssistant` will take in a `batch_request` describing data from 2019 and calculating upper and lower bounds for the following `Expectations` across the sample `Batches`. "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "id": "8362d4be",
+   "metadata": {},
+   "outputs": [
```

Comment:
I'd remove the `dict` so this looks like:
```
"outputs": [],
```
Currently it renders with a pink block this is empty error box.